.github/report-config.sample.yml

```diff
@@ -0,0 +1,7 @@
+reports:
```

Comment:
Is this file essential? Why not moving its contents to report-config.yml?

Reply:
The idea was to provide a sample. The configuration is take from [`quarkus-artemis`](https://github.com/quarkiverse/quarkus-artemis/) which has an atypical. Do you have a suggestion for a more meaningful default? I was thinking about:
```yml
reports:
  - name: Surefire
    path: '**/target/surefie-reports/TEST*.xml'
    artifact-pattern: test-report*
  - name: Failsafe
    path: '**/target/*-reports/TEST*.xml'
    artifact-pattern: test-report*
```

Reply:
That looks better, just noticed there is a typo in the surefire path ( `s/surefie/surefire`)

Reply:
How about Unit and Integration tests instead?

Reply:
Done.